src/shared/parameter_v2/config_definitions/test/example.yaml

```diff
@@ -40,23 +40,23 @@ include:
     name: example_bool_param
     value: true
     constant: true
-    description: >
+    description: >-
```

Comment:
`>` was all changed to `>-` to remove the new line characters in the description strings